src/store/reducers/iotHubManagerReducer.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import 'rxjs';
+import { Observable } from 'rxjs';
+import { IoTHubManagerService } from 'services';
+import { toDeviceListModel } from 'services/models';
+import { createReducerScenario, createEpicScenario } from 'store/utilities';
+
+// Simulation reducer constants
+const EMPTY_DEVICES = toDeviceListModel();
+const initialState = { data: undefined };
+
+// ========================= Reducers - START
+const deviceListModelReducer = (state, action) => ({ ...state, data: action.payload });
+const iotHubManagerErrorReducer = (state, action) => ({ error: action.payload });
+const initialStateReducer = (state, action) => initialState;
+
+const updateModel = { type: 'DEVICES_UPDATE', reducer: deviceListModelReducer };
+
+export const redux = createReducerScenario({
+  updateModel,
+  clearModel: { ...updateModel, type: 'DEVICES_CLEAR', staticPayload: EMPTY_DEVICES },
+  registerError: { type: 'DEVICES_ERROR', reducer: iotHubManagerErrorReducer },
```

Comment:
I'm not going to change this for this PR but I want to note that I want to change how errors are handled from simulation. In simulation I would wipe the entire reducer state to an error state to keep things easy. That worked fine given how simple the simulation reducers are but I don't think it will scale here. We can talk about this later as I'm still doing some research into the best way to approach it. The same thing applies to the clear reducer. In simulation we avoided having a reducer for loading states by hiding those semantics in `undefined` values (e.g. an  `undefined` meant the data was loading). That works ok but we should probably be more explicit about loading states too.

Reply:
Yeah, we've talked about how we might want error handling to be different.  This is really just a first pass... and there's not much here yet.  We'll figure out what we want overall... and then change to match that.